sidebars.js

```diff
@@ -96,6 +96,7 @@ module.exports = {
   },
   {
     "Okteto manifest": [
+      "manifest/environment-variables",
```

Comment:
Since they are called `Variables` in the Okteto UI, and Okteto MAnifest, I would use `Variables` for the title for consistency

Reply:
Sounds good. I'll probably merge with the `okteto-variables.mdx` page from @viddo's [latest PR](https://github.com/okteto/docs/pull/578) which addresses this feedback 👍